### DISCUSSION TITLE
[Question] is there any option  to get x,y  position when we click on the image in particular area ?

### DISCUSSION THREAD

**Discussion by santoshgistto** [hm21/pro_image_editor#86] [Q&A] [answered]
### Exmaple
<img width="704" alt="Screenshot 2024-06-13 at 10 47 24 PM" src="https://github.com/hm21/pro_image_editor/assets/12577046/aee6232e-0342-4089-b8c2-1479fd0d7732">

_No response_

**Comment by hm21** [comment]
It's not possible directly. 

- In which editor do you need this feature? And why do you require this feature?
- Do you need it only in a single editor, like only Paint-Editor, or with all editors together?

**Reply by santoshgistto** [reply]
Mostly it will be helpful if we include it in core. 
Here is the scenario  lets say user wants to generate a sticker, when tap/long press on particular object in the image.  ( similar to iOS photo gallery implementation)

**Reply by hm21** [reply, marked as answer]
In the editor version >= `4.0.0` you can now get the x,y position when you create a custom widget like below:

```dart
configs: ProImageEditorConfigs(
  customWidgets: ImageEditorCustomWidgets(
    mainEditor: CustomWidgetsMainEditor(
      bodyItems: (editor, rebuildStream) {
        return [
          ReactiveCustomWidget(
            stream: rebuildStream,
            builder: (_) {
              return Listener(
                behavior: HitTestBehavior.translucent,
                onPointerDown: (details) {
                  print({
                    'global': details.position,
                    'local': details.localPosition,
                  });
                },
              );
            },
          ),
        ];
      },
    ),
  ),
```

**Reply by santoshgistto** [reply]
thank you
